### PR TITLE
Add iconShape to checkbox

### PR DESCRIPTION
**Pull request by juanbiberretta**
#### Related issues
Fixes https://meetup.atlassian.net/browse/OS-260

#### Description
- An an optional `iconShape` prop to Checkbox component. 
- Defaults to `check` shape.

### CODE REVIEW

File: src/__snapshots__/footer.test.jsx.snap [hunk 1]
```diff
@@ -207,7 +207,7 @@ exports[`Footer exists 1`] = `
           items={
             Array [
               <span>
-                © Meetup 2018
+                © Meetup 2019
```
Comment (juanbiberretta) [inline src/__snapshots__/footer.test.jsx.snap]: lol this snapshot was failing.. thought I'd just update it

Comment (muptravis) [reply src/__snapshots__/footer.test.jsx.snap]: I thiiink we should just change the snapshot.. see here https://meetuphq.slack.com/archives/C099CDFJS/p1546441880004900

Comment (muptravis) [reply src/__snapshots__/footer.test.jsx.snap]: ^ this is jackie hi

Comment (jaclynj) [reply src/__snapshots__/footer.test.jsx.snap]: https://github.com/meetup/meetup-web-components/blob/master/src/Footer.jsx#L184 <- this is why it was failing. I think leave it for now and we'll sort it out

**Pull request merged**